config/ground-truth-yaml-files/all_features_version_10_no_visit_variant_dili.yml

```diff
@@ -497,29 +540,34 @@ patient:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id100
+    enum:
     - '0'
     - '1'
     type: string
     search_term: death
+    limit: 1
   DiabetesEndocrineDisorderDx:
     categories:
     - biolink:Disease
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id104
+    enum:
     - '0'
     - '1'
     type: string
-    search_term: diabetes, endocrine disorder
+    search_term: diabetes
+    limit: 1
+    search_term: endocrine disorder

```

Comment:
This yaml file is treated like JSON, so duplicates of the same key will overwrite each other.

Reply:
@maximusunc Good point. I did not realize this until you brought this up. @karafecho this reminded me the FHIR_mappings_pcd.yml file you created for creating FHIR_mappings output file. I verified multiple system-code specifications (e.g., for Ethnicity_UNC specification) have the later defined system-code overriden the previously defined one.

Reply:
@maximusunc : So, combined variables such as PrednisoneOrCorticosteroids_Rx and DiabetesEndocrineDisorderDx, the second search term is the more generic one, so I am fine with the second search_term overwriting the first one.

@hyi : I'm not sure I'm following your comment. In your example, there very well may be more than one ethnicity variable within a given file; however, there will only be one variable (Ethnicity_UNC) with FHIR mappings, as the other ethnicity variables will not be derived from UNC Health.

Reply:
I don't think we should be leaving it as is and relying on the overwrite to happen. In general, we shouldn't rely on object keys to stay in the same order.

Reply:
@karafecho I realized my previous comment needs some clarification. Here is the current specification in FHIR_mappings_pcd.yml file for Ethnicity_UNC variable:
```
Ethnicity_UNC:
    system: http://terminology.hl7.org/ValueSet/v3-Ethnicity
    code: 2135-2
    system: http://terminology.hl7.org/ValueSet/v3-Ethnicity
    code: 2186-5
    search_term:
    domain: Patient
    class:
```
As you can see there are two pairs of system-code specifications. I have verified in this case only the later defined 2186-5 code is used since it overwrites the previously defined 2135-2. For system, it does not matter since two specified system are the same. Does this make sense?

Reply:
I removed all dual search_terms from all_features_dili.yaml.

Thanks for the clarification, Hong. I will take a look at the file and respond in Slack, so that we can close this PR.

Reply:
@karafecho While looking into this, I found this Ethnicity_UNC variable is not included in the FHIR mappings output yaml file. Then I looked into my code and found if search_term is defined as empty, this variable specification is filtered out. I need to update my code to account for this. Otherwise, these manually defined system-code pairs are not included in the FHIR mappings output file. Do we need to rerun FHIR PIT for 2010-2016 years as well using updated FHIR mapping file? The only problem with the current FHIR mapping output file is that those Patient variables with manually defined system and code are not included.